.circleci/config.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       LANG: C.UTF-8
       FASTLANE_ITUNES_TRANSPORTER_PATH: .bundle
     docker:
-      - image: circleci/ruby:2.3
+      - image: milch/fastlane-base
```

Comment:
Best 🥛 🍼

Reply:
If you create a fastlane org on hub.docker.com we can obviously move it there, this was just more convenient for me to test 😄